packages/venia-concept/src/components/Modal/modal.js

```diff
@@ -0,0 +1,6 @@
+import { createPortal } from 'react-dom';
+
+const Modal = ({ children }) =>
+    createPortal(children, document.getElementById('root'));
```

Comment:
Do we care about keyboard focus for this iteration? If so we will need to do more work here.

See [here](https://reactjs.org/docs/portals.html#usage) where they discuss that it is important to manage keyboard focus with portals. And [here](https://reactjs.org/docs/accessibility.html#programmatically-managing-focus) where they have suggestions on how to proceed.

Aside from that, this is cool! Should we apply this to the other modals such as nav or the cart? cc @jimbo

Reply:
Hello @sirugh , thank your for your feedback. I haven't seen the keyboard focus improvements being done on other drawers.

However, I have implemented the keyboard focus on the filter modal. But I'm not really happy with the implementation. For some reason, focus wouldn't be switched to modal if the `focus()` method is not in `setTimeout` function, even with the `preventDefault` function.

You can see the implementation here: `src/RootComponents/Category/categoryContent.js`.

Do you have any suggestions on how to improve this?